pyproject.toml

```diff
@@ -17,6 +17,7 @@ pytest = "^7.2.0"
 Cython = "^0.29.32"
 ica = "^0.5"
 numba = "^0.56.4"
+KDEpy = "^1.1.0"
```

Comment:
I have recently read that this `^` syntax in Poetry is nonstandard and does not conform to PEP. Change it to regular `*`-based syntax, e.g. `KDEpy = "1.*"`. Also I think that we can loosen requirements quite a bit and fix just the major version in most cases.